pkg/agent/agent.go

```diff
@@ -255,6 +256,24 @@ func (i *Initializer) initOpenFlowPipeline() error {
 		klog.Errorf("Failed to setup openflow entries for Cluster Service CIDR %s: %v", i.serviceCIDR, err)
 		return err
 	}
+
+	go func() {
+		for {
+			if _, ok := <-ofConnCh; !ok {
+				return
+			}
+			// Give some time for the OVSDB connection to recover if necessary.
+			if ovsdbConnHealthy := i.ovsBridgeClient.CheckConnectionHealth(5 * time.Second); !ovsdbConnHealthy {
```

Comment:
@antoninbas  I see that you check the connection by getting bridge uuid every minute in this patch. I also invoke the query for OVS version to indicate OVSDB connection in my patch, see https://github.com/vmware-tanzu/antrea/pull/290/files#diff-a57e11cdb48841c4b88ef3da2324f367R292. I think we can invoke the same ovs version query so that the new query for bridge uuid can be avoided.

Reply:
Thanks for the heads-up. At the moment I have reverted my change since I think increasing the round number is not required for a reconnection. See https://github.com/vmware-tanzu/antrea/pull/297#discussion_r364402951. Let me know what you think.